runtests.sh

```diff
@@ -44,39 +44,26 @@ doQuickTests=false
 doNetTests=false
 doDryRun=false
 doUnitTests=false
-doBlackFormat=false
```

Comment:
need changes in azure pipeline (which runs everything including tests on GPU)
https://github.com/Project-MONAI/MONAILabel/blob/main/.azure/azure-build-test-gpu-pipeline.yml#L26-L37

Reply:
Fixed